main.py

```diff
@@ -0,0 +1,103 @@
+""" SHEST FASTAPI SERVER """
+
+from cryptography.fernet import Fernet
+from fastapi import FastAPI, Request
+from os import popen
+from uvicorn import run
+from fastapi.responses import StreamingResponse
+
+
+app = FastAPI()
+
+
+def load_key():
+    """Load the previously generated key"""
+
+    with open('secret.key', 'r') as file:
+        return file.read()
+
+
+def encrypt_message(message: str):
+    """
+    Encrypts a message"
```

Comment:
```suggestion
    Encrypt a message"
```

Function should document what it should do when called, not what it does. This grammar is different.

Reply:
Apply everywhere.